inabox/Makefile

```diff
@@ -26,12 +26,14 @@ deploy-all:
 	go run ./deploy/cmd  -localstack-port 4570 -deploy-resources true  all
 
 stop-infra:
-	./bin.sh stop-graph
 # Stop anvil
 	docker ps -q --filter "ancestor=ghcr.io/foundry-rs/foundry" | xargs -r docker stop 2>/dev/null || true
 # Stop localstack based on container name
 	docker ps -q --filter "ancestor=localstack/localstack" | xargs -r docker stop 2>/dev/null || true
-
+# Stop graph node stack based on container names
+	docker ps -q --filter "ancestor=graphprotocol/graph-node:v0.35.0" | xargs -r docker stop 2>/dev/null || true
+	docker ps -q --filter "ancestor=ipfs/kubo:v0.24.0" | xargs -r docker stop 2>/dev/null || true
+	docker ps -q --filter "ancestor=postgres:13" | xargs -r docker stop 2>/dev/null || true
```

Comment:
nit: we shouldnt need any of these things. when running inside test suite, they should be killed by test framework. When running via inabox, then I think its ryuk that's managing them? So we probably want to be using ryuk to GC them?

Reply:
So inabox has two modes of operation: test or devnet.

Ryuk enabled in test mode (cleanup when test ends)
Ryuk disabled in devnet mode (no cleanup, need to run make stop-infra)

It might be possible to have Ryuk enabled in devnet mode and have it do the garbage collection, will take a look in a separate diff.

Reply:
i see... i thought in inabox mode Ryuk was just outliving the test process. Makes sense then if Ryuk is just not spun up at all.